test/spec/modules/smaatoBidAdapter_spec.js

```diff
@@ -456,531 +467,534 @@ describe('smaatoBidAdapterTest', () => {
             mediaTypes: {
-              video: {
-                context: ADPOD,
-                playerSize: [[W, H]],
-                adPodDurationSec: ADPOD_DURATION,
-                durationRangeSec: DURATION_RANGE,
-                requireExactDuration: false
-              }
+              banner: BANNER_PREBID_MEDIATYPE,
+              video: VIDEO_OUTSTREAM_PREBID_MEDIATYPE
             },
-            bidId: BID_ID
+            adUnitCode: '/19968336/header-bid-tag-0',
+            transactionId: 'transactionId',
+            sizes: [[300, 50]],
+            bidId: 'bidId',
+            bidderRequestId: 'bidderRequestId',
+            src: 'client',
+            bidRequestsCount: 1,
+            bidderRequestsCount: 1,
+            bidderWinsCount: 0
           };
 
-          it('sends required fields', () => {
-            const reqs = spec.buildRequests([longFormVideoBidRequest], defaultBidderRequest);
-
-            const req = extractPayloadOfFirstAndOnlyRequest(reqs);
-            expect(req.id).to.exist;
-            expect(req.imp.length).to.be.equal(ADPOD_DURATION / DURATION_RANGE[0]);
-            expect(req.imp[0].id).to.be.equal(BID_ID);
-            expect(req.imp[0].tagid).to.be.equal(ADBREAK_ID);
-            expect(req.imp[0].bidfloor).to.be.undefined;
-            expect(req.imp[0].video.ext.context).to.be.equal(ADPOD);
-            expect(req.imp[0].video.w).to.be.equal(W);
-            expect(req.imp[0].video.h).to.be.equal(H);
-            expect(req.imp[0].video.maxduration).to.be.equal(DURATION_RANGE[1]);
-            expect(req.imp[0].video.sequence).to.be.equal(1);
-            expect(req.imp[1].id).to.be.equal(BID_ID);
-            expect(req.imp[1].tagid).to.be.equal(ADBREAK_ID);
-            expect(req.imp[1].bidfloor).to.be.undefined;
-            expect(req.imp[1].video.ext.context).to.be.equal(ADPOD);
-            expect(req.imp[1].video.w).to.be.equal(W);
-            expect(req.imp[1].video.h).to.be.equal(H);
-            expect(req.imp[1].video.maxduration).to.be.equal(DURATION_RANGE[1]);
-            expect(req.imp[1].video.sequence).to.be.equal(2);
-          });
+          const reqs = spec.buildRequests([combinedBannerAndVideoBidRequest], defaultBidderRequest);
 
-          it('sends instl if instl exists', () => {
-            const instl = { instl: 1 };
-            const bidRequestWithInstl = Object.assign({}, longFormVideoBidRequest, {ortb2Imp: instl});
+          expect(reqs).to.have.length(2);
+          expect(JSON.parse(reqs[0].data).imp[0].banner).to.deep.equal(BANNER_OPENRTB_IMP);
+          expect(JSON.parse(reqs[0].data).imp[0].video).to.not.exist;
+          expect(JSON.parse(reqs[1].data).imp[0].banner).to.not.exist;
+          expect(JSON.parse(reqs[1].data).imp[0].video).to.deep.equal(VIDEO_OUTSTREAM_OPENRTB_IMP);
+        });
 
-            const reqs = spec.buildRequests([bidRequestWithInstl], defaultBidderRequest);
+        describe('ad pod / long form video', () => {
```

Comment:
GH is not able to display the changes correctly for some reasons, but I didn't change anything in those tests, only added `if FEATURE` block here